common/djangoapps/util/views.py

```diff
@@ -56,6 +56,39 @@ def jsonable_server_error(request, template_name='500.html'):
         return server_error(request, template_name=template_name)
 
 
+def handle_500(template_path):
+    """
+    Decorator for view specific 500 error handling.
```

Comment:
I think we can put a better docstring here may be something like. "Decorator to rendering custom html template in case of error in wrapped function"